docs/operating-scylla/security/_common/upgrade-warning-runtime-authentication.rst

```diff
@@ -0,0 +1,20 @@
+
+After Upgrading from 5.4
+----------------------------
+
+The procedure described above applies to clusters where consistent topology updates 
+are enabled. The feature is automatically enabled in new clusters.
+
+If you've upgraded an existing cluster from version 5.4, ensure that you 
+:doc:`manually enabled consistent topology updates </upgrade/upgrade-opensource/upgrade-guide-from-5.4-to-6.0/enable-consistent-topology>`.
+Without consistent topology updates enabled, you must take additional steps
+to enable authentication: 
```

Comment:
The wording here suggests that the section was supposed to be applicable to enabling authentication only, but a note with a link to it is placed directly under the title "Enable and Disable Authentication Without Downtime" and the section itself is placed as a subsection of "Disable Authentication Without Downtime". It looks inconsistent to me.

IMO the wording in this section should be adjusted to explicitly cover both procedures.

Reply:
@annastuchlik